src/pybind/mgr/dashboard/frontend/src/app/ceph/block/nvmeof-gateway-group/nvmeof-gateway-group.component.html

```diff
@@ -0,0 +1,43 @@
+<ng-container *ngIf="gatewayGroup$ | async as gateways">
+  <cd-table
+    #table
+    [data]="gateways"
+    [columns]="columns"
+    columnMode="flex"
+    selectionType="single"
+    identifier="name"
+    (updateSelection)="updateSelection($event)"
+    (fetchData)="fetchData()">
+  </cd-table>
+</ng-container>
+
+<ng-template #dateTpl
+             let-events="data.value">
+  <span *ngIf="events?.length">{{ events[0].created | date:'EEE d MMM, yyyy' }}</span>
+</ng-template>
+<ng-template #gatewayTpl
+             let-gateway="data.value">
+  <div [cdsStack]="'horizontal'"
+       [gap]="'4'">
```

Comment:
```suggestion
  <div cdsStack="horizontal"
       gap="4">
```
if this is a string then this should work too for your use case.